jsapp/js/alertify.ts

```diff
@@ -93,24 +109,25 @@ export function multiConfirm(confirmId, title, message, buttons) {
   const dialog = alertify[confirmId]();
 
   // set up closing modal on ESC key
-  const killMe = (evt) => {
-    if (evt.keyCode === KEY_CODES.ESC) {
+  const killMe = (evt: JQuery.KeyUpEvent) => {
```

Comment:
Is it (or should it be) a goal to remove jquery since we also use react? This line replaces a constant with a JQuery property.

Reply:
Right, I dropped jquery usage from this file - it was being used for ESC key listening :)